tests/e2e/keycloak/01-connect.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: connect-keycloak-e2e
+spec:
+  backoffLimit: 20
+  template:
+    metadata:
+      labels:
+        e2e-test: keycloak-e2e
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: connect
+          image: docker.io/appuio/oc:v4.14
```

Comment:
This image does not work on Macs with Apple silicon: `no match for platform in manifest: not found`. Not sure why though, as other x86 images usually work with no problem.

However, as you only need curl, maybe some other image would be better suited here? Maybe some small alpine image with curl installed.

Reply:
Agree here. I also use a mac and a lighter image is welcomed.

Reply:
switched to another image also mentioned issue with that image on our #appuio-cloud RC channel